### DISCUSSION TITLE
wasmer with tiny IoT devices

### DISCUSSION THREAD

**Discussion by TAOFOR4** [wasmerio/wasmer#4635] [General]
Hi, I notice wasmer has already supported arm32 https://github.com/wasmerio/wasmer/issues/1652. But is there any examples for using wasmer on IoT devices like stm32, raspberry pico or ESP32? I haven't found any yet. Thank you!

**Comment by fehmi8** [comment]
Any answer? Thanks!

**Comment by xdoardo** [comment]
We recently added support for `wasmi` which supports `no-std` targets. It's hopefully something that does not require a lot of context, so we welcome any contribution to this effort. Please let us know if you need any guidance, [here](https://discord.gg/rWkMNStrEW)'s our discord server!
